deploy/helm/kubecf/templates/ops-use-s3-blobstore.yaml

```diff
@@ -0,0 +1,152 @@
+{{- if eq .Values.features.blobstore.provider "s3" -}}
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ops-use-s3-blobstore
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ include "kubecf.fullname" . }}
+    app.kubernetes.io/version: {{ default .Chart.Version .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ include "kubecf.chart" . }}
+data:
+  ops: |-
+    {{- .Files.Get "assets/use-s3-blobstore.yml" | nindent 4 }}
```

Comment:
Looking at this pr, while is something I would also like to have. What is this file in here, any examples on how it should look?

Reply:
The file `use-s3-blobstore.yml` is  taken from the official cloudfoundry distribution. See changes in `WORKSPACE`file.

Reply:
@kramerul so we ran this PR on our side, the above file(see https://github.com/cloudfoundry/cf-deployment/blob/master/operations/use-s3-blobstore.yml#L2-L9), does not contains a `host`, which make our `api` pod to fail, while we do not use an aws s3 endpoint, but a different one.